.Glass/glass.json

```diff
@@ -0,0 +1,143 @@
+{
```

Comment:
Does this file need to be tracked in git?

Reply:
If it is then glass will open with the same settings everytime. Glass is like a better shuffleboard that doesn't suck at graphing. I don't really care if the file gets tracked but yall should use glass anytime you need to graph something its such a better user experience.